.circleci/config.yml

```diff
@@ -136,6 +136,7 @@ jobs:
     <<: *build_and_test_android
     environment:
       PLATFORM: 1
+      TARGET: Android-ARM
```

Comment:
How did this work before?

Reply:
Because the scripts used CIRCLE_JOB which is Android-ARM. This line isn't really needed unless you recently modified behavior. :P

Reply:
Ha. Now it makes sense. Thanks for reminding me of how this works @xiaobai